packages/react-api/src/Api.tsx

```diff
@@ -186,22 +188,13 @@ function Api ({ children, store, url }: Props): React.ReactElement<Props> | null
   const [extensions, setExtensions] = useState<InjectedExtension[] | undefined>();
 
   const value = useMemo<ApiProps>(
-    () => ({ ...state, api, apiError, apiUrl: url, extensions, isApiConnected, isApiInitialized, isWaitingInjected: !extensions }),
-    [apiError, extensions, isApiConnected, isApiInitialized, state, url]
+    () => ({ ...state, api, apiError, apiUrl: apiType?.param, extensions, isApiConnected, isApiInitialized, isWaitingInjected: !extensions }),
+    [apiError, extensions, isApiConnected, isApiInitialized, state, apiType?.param]
```

Comment:
```suggestion
    [apiError, extensions, isApiConnected, isApiInitialized, state, apiType]
```

The Babel compilation always makes these a bit more hairy than it should be, so keeping to the base actually yields less suprises... :)